com.chartboost.mediation/Runtime/Events/EventProcessor.cs

```diff
@@ -21,6 +23,14 @@ internal enum FullscreenAdEvents
             Expire = 4
         }
 
+        internal enum BannerAdEvents
+        {
+            Appear = 0,
```

Comment:
Does this mean Loaded? I wonder if we want to keep the name consistent across platforms.

Reply:
Decided to use the event name as `DidLoad`